### ISSUE TITLE
Deprecation: ember-views.event-dispatcher.canDispatchToEventManager

### ISSUE THREAD

**Issue by kliput** [html-next/ember-gestures#98]
Hi,
In Ember 2.14.x app, the latest release (0.4.5) of the addon causes Ember deprecation message:
```
DEPRECATION: `canDispatchToEventManager` has been deprecated in (EventDispatcher). [deprecation id: ember-views.event-dispatcher.canDispatchToEventManager]
        at logDeprecationStackTrace (http://localhost:4200/assets/vendor.js:23913:21)
        at HANDLERS.(anonymous function) (http://localhost:4200/assets/vendor.js:24131:9)
        at raiseOnDeprecation (http://localhost:4200/assets/vendor.js:23943:14)
        at HANDLERS.(anonymous function) (http://localhost:4200/assets/vendor.js:24131:9)
        at invoke (http://localhost:4200/assets/vendor.js:24143:9)
        at deprecate (http://localhost:4200/assets/vendor.js:23997:24)
        at Class.init (http://localhost:4200/assets/vendor.js:57460:87)
        at Class.superWrapper [as init] (http://localhost:4200/assets/vendor.js:55969:22)
        at new Class (http://localhost:4200/assets/vendor.js:51167:19)
```
More info at: https://www.emberjs.com/deprecations/v2.x/#toc_custom-eventmanger-deprecated

Best regards,
Jakub

**Comment by scottkidder** [comment]
https://github.com/emberjs/rfcs/pull/194

**Comment by eriktrom** [comment]
@scottkidder - thanks 

related PR: https://github.com/html-next/ember-gestures/pull/103

I'll read up and get that fix in

**Comment by eriktrom** [comment]
closed by #103
